subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -137,8 +139,22 @@ static struct nrf_cloud_pgps_prediction *get_cached_prediction(off_t off)
 	if (prediction_buf_flash_offset != off) {
 		int err;
 
+		LOG_DBG("Checking if flash_accessible...");
```

Comment:
```c
LOG_DBG("Checking if flash_accessible...");
```
and 
```c
LOG_DBG("flash_accessible UNLOCKED");
```
Could possibly be removed (even as debug logs) without too much harm to debug-ability, something to consider

Reply:
Ditto with occurrences of `LOG_DBG("flash_accessible LOCKED");`